inst/rmd/h/_navbar.html

```diff
@@ -1,21 +1,17 @@
-
-<div class="navbar navbar-%s  navbar-fixed-top" role="navigation">
+<div class="navbar navbar-expand-%s navbar-%s bg-%s fixed-top" role="navigation">
```

Comment:
Double-check that both `4` and `4-3` look right

Reply:
Things look alright for `4`, but `4-3` has a couple small issues:

* Right menu goes off the page at a medium size
* Collapse menu appears to the left of the brand (in `3` and `4` it appears in the right hand corner)

![navbar](https://user-images.githubusercontent.com/1365941/69681075-87453f80-1072-11ea-9d8e-8834b15f1dac.gif)

Reply:
Interestingly, issue 1 goes away if I get rid of https://github.com/rstudio/bootsass/blob/master/inst/bs3compat/_navbar_compat.scss#L21 on the `.navbar-right`

Reply:
Proposed fix https://github.com/rstudio/bootsass/pull/10